helm/prometheus-rules/templates/alerting-rules/vpa.all.rules.yml

```diff
@@ -27,5 +27,5 @@ spec:
         cancel_if_scrape_timeout: "true"
         cancel_if_outside_working_hours: "true"
         severity: page
-        team: atlas
+        team: phoenix
```

Comment:
If we switch to team `phoenix`, I guess we should switch `area` to `kaas` too

Reply:
thanks, fixed